### PR TITLE
Upgrading node to the latest version for beta and unstable browsers.

### PR DESCRIPTION
**Pull request by manjeshbhargav**
@makarandp0 

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: .circleci/images/firefox/beta/Dockerfile [hunk 1]
```diff
@@ -28,9 +28,6 @@ ENV BROWSER='firefox'
 ENV BVER='beta'
 ENV FIREFOX_BIN=/opt/firefox/firefox
 
-# TODO(mmalavalli): Add iptables support once network tests are configured
```
Comment (makarandp0) [inline .circleci/images/firefox/beta/Dockerfile:31]: is this comment not applicable anymore?

**Review by makarandp0 — APPROVED** [review]
@manjeshbhargav, This change should go in master so that next iteration of browser builds will pick this up.

File: .circleci/images/firefox/beta/Dockerfile (hunk 1 above)
Comment (manjeshbhargav) [reply .circleci/images/firefox/beta/Dockerfile:31]: Yes. We already have iptables support.

**Comment by manjeshbhargav** [comment]
Closing this and re-opening as #1490 against `master`.

**Pull request closed without merging**